.github/workflows/sycl-containers.yaml

```diff
@@ -24,12 +24,31 @@ on:
 permissions: read-all
 
 jobs:
-  base_image_ubuntu2204:
+  build_and_push_images:
     if: github.repository == 'intel/llvm'
-    name: Base Ubuntu 22.04 Docker image
+    name: Build and Push Docker Images
     runs-on: ubuntu-22.04
     permissions:
       packages: write
+    strategy:
+      matrix:
+        include:
+          - name: Base Ubuntu 22.04 Docker image
+            file: ubuntu2204_base
+            tags: ubuntu2204_base
+            build_args: ""
+          - name: Build Ubuntu Docker image
+            file: ubuntu2204_build
+            tags: ubuntu2204_build
+            build_args: ""
+          - name: Intel Drivers Ubuntu 22.04 Docker image
+            file: ubuntu2204_intel_drivers
+            tags: ubuntu2204_intel_drivers
+            build_args: "use_latest=false"
+          - name: Intel Drivers (unstable) Ubuntu 22.04 Docker image
+            file: ubuntu2204_intel_drivers
+            tags: ubuntu2204_intel_drivers_unstable
```

Comment:
I think this patch changes tags we used for uploading docker images. 
@jsji, could you update the documentation https://github.com/intel/llvm/blob/sycl/sycl/doc/developer/DockerBKMs.md#sycl-containers-overview if it's needed, please?

Reply:
Yes, the tag for `unstable` is changed, but we never mentioned/use it anywhere in source tree/docs. So I don't think any updated needed. Let me know if there is some usage that I don't know.

Reply:
https://github.com/intel/llvm/pull/13307 for the image update first.